game/main.cc

```diff
@@ -209,6 +209,8 @@ void jstestLoop() {
 		config["graphic/window_width"].i() = 640;
 		config["graphic/window_height"].i() = 360;
 		Window window;
+		input::Controllers controllers;
+		controllers.enableEvents(true);
```

Comment:
As a side note i don't like this method naming/argument combi. But that's another issue on itself 😂 
`enableEvents` is already a certain action which doesn't need a bool input. A better name if you want to keep continueing the bool argument: `toggleEvents(true|false)`. Or have 2 methods: `enableEvents()` and `disableEvents()`

Reply:
Yeah we can change that later. Incidentally, I also think we should a) provide an option to configure the dead-zone for joystick axes and b) allow overriding the default in controllers.xml.

Reply:
Yeah, in looking into the issues in 769, I'm realizing that the X-plorer is pretty mis-configured. Currently the whammy bar is set to "god mode", but that should be triggered by tilting the guitar neck up. Since the gyro for that is constantly cranking out data, it would trigger no matter what. Having some kind of threshold setting would allow it to be configured properly.